### ISSUE TITLE
deepspeed.initialize() fails to complete

### ISSUE THREAD

**Issue by obackobarama** [microsoft/DeepSpeed#831]
Dear all,

I'm training a custom model on a cluster and have reserved two GPUs which are available (checked it via nvidia-smi).

I'm executing the training script via 

SEED=1635 deepspeed --num_gpus=2 train_video.py --config NETWORK_CONFIG.yaml --deepspeed_config Experiment9/ds_config.json 

However, deepspeed fails to complete the initialization, i.e. it stays there and is not able to complete the command (there are some prints in the train_video.py so that I can check which commands are completed):

![grafik](https://user-images.githubusercontent.com/76612928/110316884-a4a1e100-800b-11eb-90b7-ab45fd5917d6.png)

Any idea why this might not work out? Help is very much appreciated!

Thank you so much!

PS: I am quite new to deepspeed, so if any information for answering this question is missing: please let me know -I will provide the missing information immediately.

**Comment by obackobarama** [comment]
I also checked the tutorial on cifar 10 and get the same bug:

![grafik](https://user-images.githubusercontent.com/76612928/110327809-c060b380-801a-11eb-9f96-ba7beb009550.png)

The ds_report outputs the following:

![grafik](https://user-images.githubusercontent.com/76612928/110327906-e38b6300-801a-11eb-8251-6f52493a386c.png)

Is it possible that some packages for multi-processing are missing? After installing deepspeed via pip3 and executing either of these training routines, I was told that mpi4py is missing, which I installed afterwards.

**Comment by ConnollyLeon** [comment]
Seems like the problem happened when creating data_parallel_group. Try to initialize one in torch first?
